src/solutions/ode_solutions.jl

```diff
@@ -225,9 +226,12 @@ function (sol::AbstractODESolution)(v, t, ::Type{deriv} = Val{0}; idxs = nothing
     if t isa IndexedClock
         t = canonicalize_indexed_clock(t, sol)
     end
-    sol.interp(v, t, idxs, deriv, sol.prob.p, continuity)
+    sol(v, t, deriv, idxs, continuity)
 end
 
+# Below are many internal dispatches for different combinations of arguments to the main API
+# TODO: could use a clever rewrite, since a lot of reused code has accumulated
```

Comment:
The issue is that the symbolic dispatch is kept open, since there's more than one `issymbolic` type. We should union that in the future, but it needs https://github.com/JuliaSymbolics/SymbolicUtils.jl/issues/737